modules/environments/data.tf

```diff
@@ -0,0 +1,20 @@
+data "azuread_group" "acr" {
+  display_name = "DTS ACR Access Administrators"
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscriptions" "sds" {
+  count               = var.pipeline_environment == "prod" && var.env != "ptl" && var.env != "ptlsbox" ? 1 : 0
+  display_name_prefix = "DTS-SHAREDSERVICES-${var.env}"
+}
+
+data "azurerm_subscriptions" "sds_ptl" {
+  count               = var.pipeline_environment == "prod" && var.env == "ptl" ? 1 : 0
+  display_name_prefix = "DTS-SHAREDSERVICESPTL"
+}
+
+data "azurerm_subscriptions" "sds_ptlsbox" {
+  count               = var.pipeline_environment == "prod" && var.env == "ptlsbox" ? 1 : 0
+  display_name_prefix = "DTS-SHAREDSERVICESPTL-SBOX"
+}
```

Comment:
it would be good to clean this all up but I guess it's existing tech debt and could be looked at in a follow up PR